pkg/app/app.go

```diff
@@ -17,6 +17,25 @@ import (
 	"github.com/snyk/go-application-framework/pkg/workflow"
 )
 
+func defaultFunc_FF_CODE_CONSISTENT_IGNORES(engine workflow.Engine, config configuration.Configuration, apiClient api.ApiClient, logger *zerolog.Logger) configuration.DefaultValueFunction {
+	callback := func(existingValue interface{}) interface{} {
+		if existingValue == nil {
+			flagname := "snykCodeConsistentIgnores"
+			client := engine.GetNetworkAccess().GetHttpClient()
+			url := config.GetString(configuration.API_URL)
+			apiClient.Init(url, client)
+			result, err := apiClient.GetFeatureFlag(flagname)
```

Comment:
**Question:** Is there any retry logic implemented here, or are we introducing a point of failure here where an intermittent network failure could give a false negative?

Reply:
Currently there is no retry logic at all. The proper place for it would be in the http client.